apps/dashboard/src/hooks/use-tags-query.ts

```diff
@@ -0,0 +1,15 @@
+import { useQuery } from '@tanstack/react-query';
+import { QueryKeys } from '@/utils/query-keys';
+import { useEnvironment } from '@/context/environment/hooks';
+import { getV2 } from '@/api/api.client';
+
+export const useTagsQuery = () => {
+  const { currentEnvironment } = useEnvironment();
+  const query = useQuery<{ data: { name: string }[] }>({
+    queryKey: [QueryKeys.fetchWorkflow, currentEnvironment?._id],
+    queryFn: async () => await getV2(`/environments/${currentEnvironment!._id}/tags`),
```

Comment:
please move it to the API helper function, then we won't need to do `data.data` when using the query hook

Reply:
I don't think this is correct. If it's a `useTagsQuery` hook, then you should get back whatever was in the body, right?

Reply:
merged in order to get the import changes in. If you feel strongly about this, let's discuss privately.